core/templates/domain/utilities/language-util.service.ts

```diff
@@ -54,16 +47,24 @@ interface ContentLanguage {
   providedIn: 'root'
 })
 export class LanguageUtilService {
-  supportedAudioLanguageList: readonly AudioLanguageBackendDict[] = (
-      CONSTANTS.SUPPORTED_AUDIO_LANGUAGES);
-  autogeneratedAudioLanguageList: readonly AutogeneratedAudioLanguageDict[] = (
-      CONSTANTS.AUTOGENERATED_AUDIO_LANGUAGES);
+  get SUPPORTED_AUDIO_LANGUAGES(): typeof CONSTANTS.SUPPORTED_AUDIO_LANGUAGES {
```

Comment:
Is this standard practice for types (i.e. using "typeof X")? /cc @nishantwrp @srijanreddy98

Reply:
I don't have a strong preference, I only went with this because it's more resilient to change/because JSON files can't be assigned a type.

I've gone back to using an explicit type.

Reply:
Using typeof indicates that the only **value** that can be returned from this function is the value in`CONSTANTS.SUPPORTED_AUDIO_LANGUAGES`. Any other value with the same structure but different values will result in a typescript error. That said, I don't have any strong preference for the convention we use for return types on functions that return a constant value. So I will go with whatever @nishantwrp is fine with.

Reply:
Sg, I'm also happy to defer to @nishantwrp's decision here (or, secondarily, @vojtechjelinek's).

Reply:
I agree with @srijanreddy98, Using `typeof` here looks good since `SUPPORTED_AUDIO_LANGUAGES` is a constant and the values should ideally be the same.

Reply:
OK, thanks -- @nishantwrp, would it be possible to add this to the general instructions for types, then?

Reply:
On second thought, I was thinking why do we even need this function, like it's doing nothing but returning the constant. We can access the constant directly where this function is being used. What do you think @brianrodri ?

Reply:
I can answer this. We had to do it because the spec file was trying to test out the service with different values of the constant. Since there is no way to change the value of the constant during runtime, we decided to make this property and mock the property instead. Does that answer your question @nishantwrp.

Reply:
Oh, ok. Was reassigning the constant not possible due to typescript errors? If that's the case maybe we can use `@ts-ignore`.